Formula/sonobuoy.rb

```diff
@@ -1,8 +1,8 @@
 class Sonobuoy < Formula
   desc "Kubernetes component that generates reports on cluster conformance"
   homepage "https://github.com/vmware-tanzu/sonobuoy"
-  url "https://github.com/vmware-tanzu/sonobuoy/archive/v0.18.1.tar.gz"
-  sha256 "2443b098de78c457776ce6d13803547d33ffd563984a269be6d449886652592c"
+  url "https://github.com/heptio/sonobuoy/archive/v0.18.2.tar.gz"
```

Comment:
Why change from `vmware-tanzu` to `heption`?

Reply:
I should do in the opposite way, that is the old org reference.

Reply:
Nice catch, replace the two heptio references.